src/Views/AppListView.vala

```diff
@@ -48,6 +50,21 @@ namespace AppCenter.Views {
 
             add (scrolled);
         }
+        
+        public void set_flathub_uri (string uri) {
```

Comment:
Instead of adding a new public method, it would probably be cleaner to do:

```suggestion
        notify["current-search-term"].connect (() => {
```